### DISCUSSION TITLE
Foxglove Studio is lagging

### DISCUSSION THREAD

**Discussion by Andre-ex9** [foxglove/community#799] [Q&A]
Hello,

I am discovering Foxglove in order to use it as a visualization system for many sensors. The problem is that when i try to display at the same time an Image from a camera and the point cloud of a laser sensor (one in a 3D panel, the other in an Image one), the frame rate of the whole app goes down.

Is there a way to know if the problem comes from my computer or if there is somewhere a maximum data input for the app to run properly ?

Thanks

**Comment by foxymiles** [comment]
Can you share the data and layout you're using? Performance is an ongoing area of work for us but it helps to see specific data people are having trouble visualizing.
